src/parameterizations/vertical/MOM_geothermal.F90

```diff
@@ -392,6 +412,12 @@ subroutine geothermal_init(Time, G, param_file, diag, CS)
         x_cell_method='mean', y_cell_method='mean', area_cell_method='mean')
   if (id > 0) call post_data(id, CS%geo_heat, diag, .true.)
 
+  ! Diagnostic for tendency due to internal heat (in 3d)
+  CS%id_internal_heat_tend_3d = register_diag_field('ocean_model',&
+        'internal_heat_tend_3d', diag%axesTL, Time,             &
+        'Internal heat tendency in 3D, reveals layer(s) that heat is added to','W m-2',&
+        v_extensive = .true.)
```

Comment:
Typically no spaces around `=` for internal function arguments.

Also, this variable may need a `units=` argument?  (I will check back on this)

Reply:
Also couldn't hurt to add spaces after the commas.

Reply:
I think the positional argument 'W m-2' specifies the units, but I may be wrong.

Reply:
Yes, I think you're right.  I missed it.

Having said that, I think Bob does like the `unit=` part ot be explicit.  I guess the fact that I missed it does support that position!

Reply:
I might suggest describing this as "3D heat tendency due to internal (geothermal) sources"

Reply:
Actually I take it all back, you are right.  I don't actually see it anywhere :).

Reply:
Sorry, closed this prematurely, I missed @ashao 's suggestion.